.github/workflows/ci.yml

```diff
@@ -25,7 +25,10 @@ jobs:
           rustup target add armv7-unknown-linux-gnueabi &&
           rustup target add armv7-unknown-linux-gnueabihf &&
           rustup target add arm-unknown-linux-gnueabi &&
-          rustup target add aarch64-unknown-linux-gnu
+          rustup target add aarch64-unknown-linux-gnu &&
+          rustup target add i686-unknown-linux-gnu &&
+          rustup target add powerpc64-unknown-linux-gnu &&
+          rustup target add powerpc64le-unknown-linux-gnu
```

Comment:
Could you add x86_64-poky-linux and x86_64-poky-linux and aarch64-poky-linux

Reply:
I think those two targets exist only in the context of Yocto? Or maybe we need to point `rustup` to a different repo? Because I can't find those targets (or any `poky` targets) through my `rustup`

Reply:
Yes that's why I did not add it there but I tried manually and it worked.

Reply:
OK, thanks